README.md

```diff
@@ -141,7 +141,36 @@ This project is released under the [Apache 2.0 license](LICENSE). Some operation
 
 ## Installation
 
-Please refer to [get_started.md](docs/en/get_started.md) for installation.
+MMGeneration depends on [PyTorch](https://pytorch.org/) and [MMCV](https://github.com/open-mmlab/mmcv).
+Below are quick steps for installation.
+
+**Step 1.**
+Install PyTorch following [official instructions](https://pytorch.org/get-started/locally/), e.g.
+
+```python
+conda create -n open-mmlab python=3.7 -y
```

Comment:
Maybe we should just use pip to install, since there are users who don't use conda.